odata-protocol/11.4 Data Modification.md

```diff
@@ -654,20 +672,22 @@ operations.
 On failure, the service MUST NOT apply any of the changes specified in
 the request.
 
-### ##subsubsec Upsert an Entity
+#### ##subsubsubsec Upsert an Entity
```

Comment:
Given that the Update section requires failing the request when the addressed entity does not exist, I find it confusing to make this now a subsection.